frontend/src/employee-frontend/components/IncomeStatementPage.tsx

```diff
@@ -292,30 +348,101 @@ function AccountantInfo({ accountant }: { accountant: Accountant }) {
   )
 }
 
-function Attachments({ attachments }: { attachments: Attachment[] }) {
+function CitizenAttachments({ attachments }: { attachments: Attachment[] }) {
   const { i18n } = useTranslation()
   return (
     <>
-      <H2>{i18n.incomeStatement.attachmentsTitle}</H2>
+      <H2>{i18n.incomeStatement.citizenAttachments.title}</H2>
       {attachments.length === 0 ? (
-        <p>{i18n.incomeStatement.noAttachments}</p>
+        <p>{i18n.incomeStatement.citizenAttachments.noAttachments}</p>
       ) : (
         <Row
           label={`${i18n.incomeStatement.attachments}:`}
-          value={attachments.map((attachment) => (
-            <FileDownloadButton
-              key={attachment.id}
-              file={attachment}
-              fileFetchFn={getAttachmentBlob}
-              onFileUnavailable={() => undefined}
-            />
-          ))}
+          value={<UploadedFiles files={attachments} />}
         />
       )}
     </>
   )
 }
 
+function UploadedFiles({ files }: { files: Attachment[] }) {
+  return (
+    <FixedSpaceColumn>
+      {files.map((file) => (
+        <div key={file.id}>
+          <FileIcon icon={fileIcon(file)} />
+          <FileDownloadButton
+            file={file}
+            fileFetchFn={getAttachmentBlob}
+            onFileUnavailable={() => undefined}
+          />
+        </div>
+      ))}
+    </FixedSpaceColumn>
+  )
+}
+
+const FileIcon = styled(FontAwesomeIcon)`
+  color: ${(p) => p.theme.colors.main.primary};
+  margin-right: ${defaultMargins.s};
+`
+
+function EmployeeAttachments({
+  incomeStatementId,
+  attachments,
+  onUploaded,
+  onDeleted
+}: {
+  incomeStatementId: UUID
+  attachments: Attachment[]
+  onUploaded: (attachment: Attachment) => void
+  onDeleted: (id: UUID) => void
+}) {
+  const { i18n } = useTranslation()
+
+  const handleUpload = React.useCallback(
+    async (
+      file: File,
+      onUploadProgress: (progressEvent: ProgressEvent) => void
+    ) => {
+      return (
+        await saveIncomeStatementAttachment(
```

Comment:
These handlers might be a little bit more readable with the curly braces removed. 🤷 `=> (await ..).map(..)`

Reply:
Fixed